types/params_test.go

```diff
@@ -209,27 +209,27 @@ func TestConsensusParamsValidation(t *testing.T) {
 			params: makeParams(makeParamsArgs{
 				blockBytes:   -2,
 				evidenceAge:  2,
-				precision:    1,
-				messageDelay: 1,
+				precision:    time.Nanosecond,
+				messageDelay: time.Nanosecond,
 			}),
 			valid: false,
 		},
-		// test invalid pubkey type provided
+		// test invalid synchrony params
 		{
-			name: "messageDelay -1",
+			name: "messageDelay 0",
 			params: makeParams(makeParamsArgs{
 				evidenceAge:  2,
-				precision:    1,
-				messageDelay: -1,
+				precision:    time.Nanosecond,
+				messageDelay: 0,
```

Comment:
Why this change?

Reply:
0 should be invalid

Reply:
Then, can you test both, 0 and -1? (i.e., add an extra TC here)

Reply:
Ok.

Reply:
🙏